app/controllers/coronavirus/pages_controller.rb

```diff
@@ -1,7 +1,6 @@
 module Coronavirus
   class PagesController < ApplicationController
     before_action :require_coronavirus_editor_permissions!
-    before_action :redirect_to_index_if_slug_unknown, only: %w[show]
```

Comment:
This seems good 👍  I don't find it _too_ scandalous 😆  Seems weird we'd give these pages a special redirect handling.

Reply:
The initialisation still happens when the index page is loaded. That still uses `ModelBuilder` and hasn't been removed. It's called by the `initialise_pages` before action.